src/index.spec.ts

```diff
@@ -1179,6 +1139,50 @@ describe('EppoPrecomputedJSClient E2E test', () => {
   });
 });
 
+describe('offlinePrecomputedInit', () => {
+  let mockLogger: IAssignmentLogger;
+  let precomputedConfigurationWire: string;
+
+  beforeAll(() => {
+    precomputedConfigurationWire = readMockPrecomputedResponse(MOCK_PRECOMPUTED_RESPONSE_FILE);
```

Comment:
Currently, tests use the unobfuscated, unstringified `ConfigurationWire` format https://github.com/Eppo-exp/sdk-test-data/blob/8ee55117b7c7f5d02b2997629af0f0fa93e4e97c/configuration-wire/precomputed-v1.json

In a follow up PR I will add support for parsing json strings and handling obfuscated fields